packages/orbit-components/src/InputField/types.d.ts

```diff
@@ -51,6 +51,7 @@ export interface Props extends Common.Globals, Common.SpaceAfter, Common.DataAtt
   readonly onKeyDown?: React.KeyboardEventHandler<HTMLInputElement>;
   readonly onKeyUp?: React.KeyboardEventHandler<HTMLInputElement>;
   readonly ariaLabel?: string;
+  readonly ariaLabelledBy?: string;
```

Comment:
Uff we are not being consistent here 😬 some are `ariaLabelledby` and some are `ariaLabelledBy`. Given that the attribute is `aria-labelledby`, we should follow the `ariaLabelledby` convention. So please change it here